src/index.js

```diff
@@ -49,19 +49,22 @@ function parse(model, response, options = {}) {
 export default Bookshelf => {
   const Model = Bookshelf.Model.prototype;
   const client = Bookshelf.knex.client.config.client;
-  const parseOnFetch = client === 'sqlite' || client === 'sqlite3';
+  const handleOnSaving = client !== 'mysql';
```

Comment:
For what it's worth, my suggestion would be to simply remove `handleOnSaving` and the associated logic for it.  In my experiences so far, that gets everything working perfectly.